src/app/shell/admin-tools/data/admins/create-admin/create-admin.component.ts

```diff
@@ -0,0 +1,137 @@
+import { ValidationConstants } from 'src/app/shared/constants/validation';
+import { AdminRole } from 'src/app/shared/enum/admins';
+import { NavBarName } from 'src/app/shared/enum/navigation-bar';
+import { Role } from 'src/app/shared/enum/role';
+import { Institution } from 'src/app/shared/models/institution.model';
+import { NavigationBarService } from 'src/app/shared/services/navigation-bar/navigation-bar.service';
+import { GetAllInstitutions } from 'src/app/shared/store/meta-data.actions';
+import { MetaDataState } from 'src/app/shared/store/meta-data.state';
+import { AddNavPath, DeleteNavPath } from 'src/app/shared/store/navigation.actions';
+import { RegistrationState } from 'src/app/shared/store/registration.state';
+import { Util } from 'src/app/shared/utils/utils';
+import { Location } from '@angular/common';
+import { CreateFormComponent } from 'src/app/shell/personal-cabinet/shared-cabinet/create-form/create-form.component';
+import { ConfirmationModalWindowComponent } from 'src/app/shared/components/confirmation-modal-window/confirmation-modal-window.component';
+import { ModalConfirmationType } from 'src/app/shared/enum/modal-confirmation';
+import { MinistryAdmin } from 'src/app/shared/models/ministryAdmin.model';
+import { CreateMinistryAdmin } from 'src/app/shared/store/admin.actions';
+import { CreateAdminTitle } from 'src/app/shared/enum/enumUA/tech-admin/create-admin';
+
+const defaultValidators: ValidatorFn[] = [
+  Validators.required, 
+  Validators.pattern(NAME_REGEX),
+  Validators.minLength(ValidationConstants.INPUT_LENGTH_1),
+  Validators.maxLength(ValidationConstants.INPUT_LENGTH_60)
+]
+@Component({
+  selector: 'app-create-admin',
+  templateUrl: './create-admin.component.html',
+  styleUrls: ['./create-admin.component.scss']
+})
+export class CreateAdminComponent extends CreateFormComponent implements OnInit, OnDestroy {
+  readonly validationConstants = ValidationConstants;
+  readonly phonePrefix = Constants.PHONE_PREFIX;
+  readonly mailFormPlaceholder = Constants.MAIL_FORMAT_PLACEHOLDER;
+  readonly adminsRole = AdminRole;
+  readonly title = CreateAdminTitle;
+  
+  @Select(MetaDataState.institutions)
+  institutions$: Observable<Institution[]>;
+  
+  AdminFormGroup: FormGroup;
+  adminRole: AdminRole;
+
+  constructor(
+    store: Store,
+    route: ActivatedRoute,
+    navigationBarService: NavigationBarService,
+    private formBuilder: FormBuilder,
+    private matDialog: MatDialog,
+    private location: Location
+  ) { 
+    super(store, route, navigationBarService);
+
+    this.AdminFormGroup = this.formBuilder.group({
+      lastName: new FormControl('', defaultValidators),
+      firstName: new FormControl('', defaultValidators),
+      middleName: new FormControl('', [
+        Validators.pattern(NAME_REGEX),
+        Validators.minLength(ValidationConstants.INPUT_LENGTH_1),
+        Validators.maxLength(ValidationConstants.INPUT_LENGTH_60)
```

Comment:
I see these validators are absolutely the same as defaultValidators, so I belive you can reuse them